addons/api/addon/serializers/session-recording.js

```diff
@@ -11,4 +11,26 @@ export default class RecordingSerializer extends ApplicationSerializer.extend(
   attrs = {
     connection_recordings: { embedded: 'always' },
   };
+
+  serialize() {
+    let serialized = super.serialize(...arguments);
+    // the post request doesn't expect a payload, so we just delete these
```

Comment:
Should this be doing a check for `reapply-storage-policy` and then we just pull out the fields we actually need instead?